ci-operator/jobs/openshift/release/openshift-release-master-periodics.yaml

```diff
@@ -27666,6 +27666,81 @@ periodics:
     - name: result-aggregator
       secret:
         secretName: result-aggregator
+- agent: kubernetes
+  cluster: build02
+  cron: 30 */6 * * *
+  decorate: true
+  decoration_config:
+    skip_cloning: true
+  extra_refs:
+  - base_ref: master
+    org: openshift
+    repo: release
+  labels:
+    ci-operator.openshift.io/prowgen-controlled: "true"
+    ci-operator.openshift.io/variant: nightly-4.9
+    job-release: "4.9"
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-openshift-release-master-nightly-4.9-e2e-aws-workers-rhel8
+  spec:
```

Comment:
This is probably optional true as well?

Reply:
The periodics don't have the flag `optional: true` so I don't think it is necessary here.